jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,9 @@
+.level = INFO
+io.netty.level = INFO
+io.gcr.level = INFO
```

Comment:
How would an explicit filter work so that it deals with the Netty background threads?